### ISSUE TITLE
Wrong output for custom images

### ISSUE THREAD

**Issue by liuhuihuii** [rishiswethan/Cancer-detection-using-CNN#7]
I solved the problem with the image size last night and I should thank you for that, however, after I trained the network for 24 epochs, the prediction of custom images are still far from correct. The percentage of each label it gives for custom images is around 25%, while testing random images from the data set gives very definite answer (50% for one label). Is it because of overfitting or not enough epochs? Is there any advice to improve the correctness for custom images?

**Comment by rishiswethan** [comment]
I originally set the epoch count to 10, maybe try that

**Comment by liuhuihuii** [comment]
But when I use your pretrianed mode 'my_model3.h5' t to predict my own cunstom image named 'InvasiveC2_2048x1536.jpg' ,then output is like:
Average from all crops

Benign : 20.6134%
InSitu : 31.7345%
Invasive : 20.0719%
Normal : 27.5802%

**Comment by rishiswethan** [comment]
Not sure why, are these from your images?

**Comment by liuhuihuii** [comment]
yep

**Comment by liuhuihuii** [comment]
I downloaded them from the internet. Maybe they are not labeled correctly, but I trust wiki anyways

**Comment by liuhuihuii** [comment]
For example, this image is from Wikipedia and it says normal on the site
the output is
Benign : 29.1999%
InSitu : 21.6953%
Invasive : 26.8959%
Normal : 22.2089%
all percentage is close to 25%
![normalc1_2048x1536](https://user-images.githubusercontent.com/40494837/50543943-59b50380-0c21-11e9-9e23-be8008375b96.jpg)

**Comment by rishiswethan** [comment]
Maybe there is some mismatch in the image formats then, I'm not sure what's
wrong
